diff --git a/node_modules/nan/nan_typedarray_contents.h b/node_modules/nan/nan_typedarray_contents.h
index f2650ed..850760e 100644
--- a/node_modules/nan/nan_typedarray_contents.h
+++ b/node_modules/nan/nan_typedarray_contents.h
@@ -34,7 +34,7 @@ class TypedArrayContents {
 // Actually it's 7.9 here but this would lead to ABI issues with Node.js 13
 // using 7.8 till 13.2.0.
 #if (V8_MAJOR_VERSION >= 8)
-      data = static_cast<char*>(buffer->GetBackingStore()->Data()) + byte_offset;
+      data = static_cast<char*>(buffer->Data()) + byte_offset;
 #else
       data = static_cast<char*>(buffer->GetContents().Data()) + byte_offset;
 #endif
